scss/_list-group.scss

```diff
@@ -83,6 +83,14 @@
       border-top-width: $list-group-border-width;
     }
   }
+  
+  // Make single `.list-group-item-action`s in `.list-group-item` fill their container
+  .list-group-item-action:only-child {
```

Comment:
It seems unlikely that this will always be an only child. Hesitant to try to solve this problem when the mentioned issue is also already closed.

Reply:
Me Me go

Reply:
Me Me go

Reply:
@mdo Based on the description, I guess it's meant to only targets the "only direct child" case.

Reply:
@mdo Indeed, the `:only-child` selector might cause issues, with invisible content mostly (`.sr-only`, `<span hidden>`, or `<input type="hidden">` come to mind). It would be acceptable to remove `:only-child` from the selector. Since `.list-group-item-action` do not display properly when inside `.list-group-item` I don't think they have been used, which means little chance of breaking backwards-compatibility.
More complex markup would still require using flexbox magic, as explained later in the docs.

As it stands, I believe that mixing list-items with plain text, complex layout and full-line links is quite a frequent use case. This 4-line hack-free addition would allow mixing links and text in list-groups, like in dropdowns.

If you or other maintainers believe this isn't worth it, just close the PR and I'll move on with absolutely no hard feelings. Otherwise I can rebase, lint, and include an example in the documentation, and wrap it all up with a nice red ribbon. 🎁 😉